### ISSUE TITLE
a11y:  Page title version ID does not have enough color contrast

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9583]
### Q&A (please complete the following information)
 - OS: [e.g. macOS] Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]  Swagger UI React 5.11.3 
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic/default

### Describe the bug you're encountering
We are using a tool called Sort site to evaluate our site for 508 accessibility.  It is flagging that there is not enough color contrast in the version identifier at the top of the page.  The Sort site error is below.  
Violates WCAG 2.0 AA 1.4.3  https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
This is on the "pre" with class "version", which is contained in the h2 with class "title".

### To reproduce...
n/a

### Expected behavior
Color should be darkened

### Screenshots
Here's the element with the error (the 1.0.0)
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/110ab0b0-0743-4466-b0ce-6fa58ff2b1a5)

Here's the Sort Site information:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/0057c266-74f3-4d02-999c-620f4f905b1f)

**Comment by dafeder** [comment]
@char0n possible to get the a11y label added to this and the related PR #9703?
